.binder/environment-python_and_r.yml

```diff
@@ -90,3 +89,7 @@ dependencies:
   - r-sp
   - r-xtractomatic
   - r-xts
+platforms:
+  - linux-64
+  - osx-64
+  - win-64
```

Comment:
We only build the lockfile for the main env and for these 3 platforms. We are missing ppcle, aarch64, and osx-arm64, only the last two may be worth adding here and it should be possible in the future when the migration we started in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4897 finishes.